yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
+-- > get HomeR
+--
```

Comment:
This example would make more sense if there was no empty line. Right now `get HomeR` and `followRedirect` are appearing as separate code blocks, as if they were separate examples or something